src/Infrastructure/Persistence/Configurations/BaseConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using CleanArchitecture.Domain.Common;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace CleanArchitecture.Infrastructure.Persistence.Configurations
+{
+    public class BaseConfiguration<TEntity> : IEntityTypeConfiguration<TEntity> where TEntity : CommonEntity
+    {
+        public virtual void Configure(EntityTypeBuilder<TEntity> builder)
+        {
+           // builder.Property<int>(x => x.Id).HasColumnName(@"Id").IsRequired(true).UseIdentityColumn().ValueGeneratedOnAdd();
```

Comment:
Better not to check in commented out code, it doesn't typically add value and can cause confusion. Why is this here? Is it important? Can I remove it or does someone else need it?